packages/lisk-p2p/src/peer.ts

```diff
@@ -112,21 +110,6 @@ const convertNodeInfoToLegacyFormat = (
 	nonce: nodeInfo.options ? (nodeInfo.options.nonce as string) : '',
 });
 
-const convertLegacyPeerInfoToNewFormat = (
```

Comment:
maybe we can add this function in `validation.ts`?

Reply:
Sounds good. Maybe this could it be done as a separate issue? I didn't want to change the existing validation function and risk breaking it.

Reply:
Yes, it's a good idea to do it in a separate PR.

Reply:
@jondubois Please make another issue regards to this